openshift/ftest-openshift-assistant-operational-methods/src/test/java/org/arquillian/cube/openshift/operational/methods/OpenShiftAssistantOperationalMethodsIT.java

```diff
@@ -47,4 +52,29 @@ public void should_apply_template_programmatically() throws IOException {
         assertThat(response.code()).isEqualTo(200);
         assertThat(response.body().string()).isEqualTo("Hello from Arquillian Template\n");
     }
+
+    @Test
+    public void should_list_all_projects() {
+        String currentProject = openShiftAssistant.getCurrentProject();
+
+        List<Project> projects = openShiftAssistant.listProjects();
+
+        assertThat(projects).extracting(Project::getMetadata)
+            .hasSize(2)
+            .extracting(ObjectMeta::getName).contains(currentProject);
+    }
+
+    @Test
+    public void should_find_project() {
```

Comment:
When I was reading this method I thought more about a method that finds a project by name rather than just returns the current project, so I think that the test should be renamed to something like `should_return_current_project`

Reply:
It actually finds a project by name.
> assertThat(openShiftAssistant.findProject(currentProject)).isPresent()

Reply:
Ah right I see what you mean now. Then separate the test with given/when/then blocks might help.